cxx-qt-lib/src/qvariant.rs

```diff
@@ -7,19 +7,16 @@
 // We are only using references to QVariant so it is actually ffi safe as far as we are concerned
 #![allow(improper_ctypes)]
 
-use crate::actually_private::Private;
-use cxx::{type_id, ExternType};
+use cxx::{memory::UniquePtrTarget, type_id, ExternType};
 use std::{
     ffi::c_void,
     marker::{PhantomData, PhantomPinned},
-    mem::{self, ManuallyDrop, MaybeUninit},
-    ops::{Deref, DerefMut},
-    pin::Pin,
+    mem::MaybeUninit,
 };
 
 #[repr(u8)]
 #[allow(dead_code)]
-enum QVariantType {
+enum QVariantEnum {
```

Comment:
I think we should stick with the "QVariantType" name here.
Having an enum named "Enum" doesn't really add value to the name

Reply:
Done